### ISSUE TITLE
HTTPS cert expired

### ISSUE THREAD

**Issue by dm3ch** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#3926]
On the https://www.googleplaymusicdesktopplayer.com/ https cert have expired 27 March

**Comment by ChristianWilkie** [comment]
yep came here to say the same thing

![image](https://user-images.githubusercontent.com/7551271/114323234-47112080-9af2-11eb-84f7-99871403deee.png)

**Comment by martin-braun** [comment]
Still not fixed. Very disappointed. Looks like the dev cannot or doesn't want to take care of the project anymore.

**Comment by LurkerShark** [comment]
Same issue as seen previously https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/issues/3789 

That issue was closed when I mentioned it in 2020 without a proper fix 🙂

**Comment by dm3ch** [comment]
@LurkerShark I'm not sure that it's completely same problem.

If I right understood in https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/issues/3789 the answer from maintainers was - oops cron haven't performed in right time due to business of server.

But now it's September and cert have been expired in March, so I assume that right now cert renewal cron is just disabled... 

P.S. Anyway unfortunately this project now looks completely dead. I assume that it was left without maintenance cause its main focus was Google Play Music and now Google Play Music is closed.

**Comment by LurkerShark** [comment]
@dm3ch that's the thing, there's no situation under #3789 where it shouldn't have run weeks before the certificate expired

I believe it was manually run after my original post back then

> P.S. Anyway unfortunately this project now looks completely dead. I assume that it was left without maintenance cause its main focus was Google Play Music and now Google Play Music is closed.

Yeah, seems that way

**Comment by ddodogames** [comment]
yeah also looks like google chrome starts thinking that lts unsafe 
![image](https://user-images.githubusercontent.com/69091682/136671534-c8d56b5f-fd79-4766-aa24-a3028b9deac0.png)

**Comment by chuckmitchell** [comment]
I guess this project is abandoned?
